tutorials/tour/_posts/2017-02-13-classes.md

```diff
@@ -9,46 +9,103 @@ categories: tour
 num: 4
 next-page: traits
 previous-page: unified-types
+topics: classes
+assumed-knowledge: no-return-keyword, type-declaration-syntax, string-interpolation, procedures
 ---
 
-Classes in Scala are static templates that can be instantiated into many objects at runtime.
-Here is a class definition which defines a class `Point`:
+Classes in Scala are blueprints for creating objects. They can contain methods,
+values, and variables, which are collectively called _members_.
+
+## Defining a class
+A minimal class definition is simply the keyword `class` and
+an identifier:
```

Comment:
It's important to note that class names should be capitalized because it's a universal convention and it has a downstream effect on pattern-matching.

*A minimal class definition is simply the keyword class and an identifier. Class names should be capitalized.*